core/input/generisdata_tech.prn

```diff
@@ -155,14 +155,14 @@ eta                 1.00        1.00        1.00
 omf                 0.02        0.02        0.02
 lifetime              45          45          45
 
-+             weathering         dac
++             weathering         dac      oae_ng      oae_el
 tech_stat                          4
-inco0               0.01       18800
-mix0                   0           0
-eta                 1.00        1.00
-omf                 0.84       0.025
+inco0               0.01       18800         200         400
```

Comment:
Should inco0 not be converted to $/tC from $/tCaO as calculated below? And why is inco0 for DACCS a factor of 100 higher?

Reply:
It should, but this inco0 is converted inside the code as the cost USD/tC depends on efficiency which a user can set. The capex is only 15$/tCaO, hence the difference between OAE and DACCS. You can see the calculations below in the same file.

Reply:
could you give an example of an efficiency, or even better the whole range we use, so that one can follow these conversions and compare them with DAC?